ostree/Docker/entrypoint.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Copyright (c) 2019, Arm Limited and Contributors. All rights reserved.
```

Comment:
Same here, copyright 2021 Pelion.

Reply:
Should be also Apache 2.0 (generically, all of these) - top level header file on this repo is Apache 2.0 license. No BSD-3 ways @petedyerarm.